src/Console/Command/ShowCommand.php

```diff
@@ -0,0 +1,507 @@
+    private $undefinedFixers;
+
+    /** @var array */
+    private $fixerList = [];
+
+    /** @var bool */
+    private $hideConfigured;
+
+    /** @var bool */
+    private $hideEnabled;
+
+    /** @var bool */
+    private $hideRisky;
+
+    /** @var bool */
+    private $hideInherited;
+
+    /** @var bool */
+    private $hideDeprecated;
+
+    /** @var bool */
+    private $hideCustom;
+
+    /** @var bool */
+    private $hideInheritance;
+
+    /** @var int */
+    private $countConfiguredFixers = 0;
+
+    /** @var int */
+    private $countRiskyFixers = 0;
+
+    /** @var int */
+    private $countEnabledFixers = 0;
+
+    /** @var int */
+    private $countInheritedFixers = 0;
+
+    /** @var int */
+    private $countDeprecatedFixers = 0;
+
+    /** @var int */
+    private $countCustomFixers = 0;
+
+    /**
+     * @param null|FixerFactory $fixerFactory
+     */
+    public function __construct(ToolInfoInterface $toolInfo, FixerFactory $fixerFactory = null)
+    {
+        parent::__construct();
+
+        $this->toolInfo = $toolInfo;
+
+        if (null === $fixerFactory) {
+            $fixerFactory = new FixerFactory();
+            $fixerFactory->registerBuiltInFixers();
+        }
+
+        $this->fixerFactory = $fixerFactory;
+
+        $this->fixerNameValidator = new FixerNameValidator();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName(self::COMMAND_NAME)
+            ->setDefinition(
+                [
+                    new InputOption('config', '', InputOption::VALUE_REQUIRED, 'The path to a .php_cs file.'),
+                    new InputOption('hide-configured', '', InputOption::VALUE_NONE, 'Hide fixers that are configured (in the config file or because of inheritance).'),
+                    new InputOption('hide-enabled', '', InputOption::VALUE_NONE, 'Hide fixers that are currently enabled (the ones that are not disabled with [\'fixer_name\' => false]).'),
+                    new InputOption('hide-risky', '', InputOption::VALUE_NONE, 'Hide fixers that are marked as risky.'),
+                    new InputOption('hide-inherited', '', InputOption::VALUE_NONE, 'Hide fixers that inherited from RuleSets.'),
+                    new InputOption('hide-deprecated', '', InputOption::VALUE_NONE, 'Hide fixers that are deprecated.'),
+                    new InputOption('hide-custom', '', InputOption::VALUE_NONE, 'Hide fixers that are custom.'),
+                    new InputOption('hide-inheritance', '', InputOption::VALUE_NONE, 'Hide the addition inheritance information.'),
+                    new InputOption('compare', '', InputOption::VALUE_NONE, 'Dumps the comparing result between your config and all available fixers in a copy-and-pastable format ready for the .php_cs file.'),
+                ]
+            )
+            ->setDescription('Shows existent Fixers with the ones actually configured or enabled by inheritance.')
```

Comment:
I would write _"Lists all available fixers and shows which ones are enabled, inherited or disabled."_.

Reply:
Done